### ISSUE TITLE
Allow closing modal using ESC key

### ISSUE THREAD

**Issue by earnjam** [bluehost/endurance-wp-module-business-reviews#1]
We should be able to close the modal window without having to tab to the close button in the corner to improve accessibility.
